Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -448,6 +448,11 @@ def audit_options
         problem "Use '--with#{$1}-test' instead of '--#{o.name}'. Migrate '--#{o.name}' with `deprecated_option`."
       end
     end
+
+    return unless @new_formula
+    if formula.deprecated_options.size
```

Comment:
`formula.deprecated_options.size` is always true.
Probably this is better: `unless formula.deprecated_options.empty?`